index.js

```diff
@@ -15,4 +15,23 @@ async function start() {
   console.log('You entered: ' + secretNumber);
   // Now try and complete the program.
   process.exit();
-}
+
+  async function start() {
+  while (1) {
```

Comment:
This will cause an infinite loop with no escape even when you update 'done' since you're not evaluating 'done' you're evaluating 1 which is always truthy.